src/webgpu/api/validation/queue/submit.spec.ts

```diff
@@ -45,3 +45,18 @@ g.test('command_buffer,device_mismatch')
       t.device.queue.submit([cb0, cb1]);
     }, mismatched);
   });
+
+g.test('command_buffer,duplicate_buffers')
+  .desc(
+    `
+    Tests submit cannot be called with the same command buffer listed multiple times:
+    `
+  )
+  .fn(t => {
+    const encoder0 = t.device.createCommandEncoder();
+    const cb = encoder0.finish();
+
+    t.expectValidationError(() => {
+      t.device.queue.submit([cb, cb]);
```

Comment:
Should test whether `cb` is invalid after this or not. (Your spec PR would say that it is still valid.)

Reply:
Hm... looks like we're still figuring that one out? According to https://bugs.chromium.org/p/dawn/issues/detail?id=1863 anyway. (Dawn invalidates them whether validation passes or not: https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/src/dawn/native/Queue.cpp;l=285;drc=f0f5f3ceebb00da9363ccc7a1e2c0f17b6b383ba)

Reply:
OK, we can test it afterward. I dunno why I didn't file a spec issue about it?

Reply:
https://github.com/gpuweb/gpuweb/issues/4599